policies/accepted/0001.md

```diff
@@ -0,0 +1,103 @@
+# 0001: Policies about Policies
+
+## Rationale
+
+We have a diverse group of people with varying ideas, availability, and background.
+By agreeing to some basic policies, and ensuring they are codified and well documented,
+we can ensure everyone working on the project is aware of the intended way of doing certain things.
+
+By having a clear set of guidelines, people are empowered to work on things with confidence that
+they are doing things in a way consistent with the goals of the project.
+
+## What this isn't
+
+We shouldn't make overly restrictive policies aimed toward contributors.
+
+These policies should serve to guide the direction of new features,
+how those new features are added, how releases work, and how we plan to interact with
+the community of cog creators.
+
+If a policy has an impact on the average new contributor,
+it needs to be in the issue/PR templates. 
+Policies should serve to clear the way to do things with more clarity on direction,
+not serve as only obstacles to progress.
+
+No policy should be considered final.
+Just because a policy was rejected,
+doesn't mean it can't be reviewed later if circumstances change.
+Any policy can be reviewed at a later date by another policy.
+
+## How policies should be made
+
+Policies have an impact on everyone who uses Red, but the largest impact is on those who maintain Red.
+These are the guidelines we use when making decisions beyond just "Does this work?"
+
+Policies should be made, and left open for discussion before anything else.
+
+While it may require a vote, concerns brought up within a reasonable window should be addressed. Concerns which come up after a policy has already been decided on may lead to a revision if deemed neccessary.
+
+A policy may be accepted at any point which there are enough votes for acceptance.
+
+A policy may be accpeted provisionally by default if open with some support and no objections after an acceptable time frame.
+
+## Voting process
+
+* Everyone in the cog creators organization has a direct vote.
+* If a policy is about one of the following, specific teams have their vote elevated in importance
+  * Policies about Red's code:
+    - Veto Power: Core Developers
+    - Additional Requirements:
+      - 1 or More Core Developers in favor
```

Comment:
This is formatted differently than in the rest of mentioned policy types (bullet list of requirements vs  just one requirement specified after colon)